README.md

```diff
@@ -381,54 +381,54 @@ Built-in functions. Ready to be bound to any key you like. :smile:
 
 ### LSP Pickers
 
-| Functions                           | Description                                                                                 |
-|-------------------------------------|---------------------------------------------------------------------------------------------|
-| `builtin.lsp_references`            | Searches in LSP references.                                                                 |
-| `builtin.lsp_document_symbols`      | Searches in LSP Document Symbols in the current document.                                   |
-| `builtin.lsp_workspace_symbols`     | Searches in LSP all workspace symbols.                                                      |
-| `builtin.lsp_code_actions`          | Lists LSP action to be trigged on enter.                                                    |
-| `builtin.lsp_range_code_actions`    | Lists LSP range code action to be trigged on enter.                                         |
-| `builtin.lsp_document_diagnostics`  | Lists LSP Diagnostics in the current document.                                              |
-| `builtin.lsp_workspace_diagnostics` | Lists LSP Diagnostics in the workspace if supported and otherwise open buffers.             |
-| `builtin.lsp_definitions`           | Goto definition if there is only one. If there are multiple, open them up in telescope     |
+| Functions                                   | Description                                                                                 |
+|---------------------------------------------|---------------------------------------------------------------------------------------------|
+| `builtin.lsp_references`                    | Searches in LSP for references                                                              |
+| `builtin.lsp_document_symbols`              | Searches for LSP document symbols in the current buffer                                     |
+| `builtin.lsp_workspace_symbols`             | Searches for LSP document symbols in the current workspace                                  |
+| `builtin.lsp_dynamic_workspace_symbols`     | Searches LSP for all workspace symbols asynchronously                                       |
+| `builtin.lsp_code_actions`                  | Lists any LSP actions for the current line, that can be triggered with <cr>                 |
+| `builtin.lsp_range_code_actions`            | Lists any LSP actions for a certain range, that can be triggered with <cr>                  |
+| `builtin.lsp_document_diagnostics`          | Searches LSP diagnostics in the current buffer                                              |
+| `builtin.lsp_workspace_diagnostics`         | Searches LSP diagnostics in the workspace if supported, otherwise searches in open buffers  |
+| `builtin.lsp_definitions`                   | Goto the definition if there's only one, otherwise show all options in Telescope            |
```

Comment:
Lists .... sounds better then searches.

Reply:
I'll rewrite all of those and have you look at the README with my next commit again with using "Lists" instead to reword those :)

Reply:
Most recent commit should have updates for this particular note, updated the README with these, and all the LSP README updates should also now be reflected in `builtin/lsp.lua` function headers too